client/src/components/NavBar.js

```diff
@@ -17,7 +17,7 @@ import MenuIcon from '@material-ui/icons/Menu';
 
 import SvgProductLaunchLogo from './ProductLaunchLogo';
 import NotificationDropdown from './NotificationDropdown';
-import { messages } from '../dummyData/dropDownItems'; //standin data
+import { getNotifications } from '../api/notifications';
```

Comment:
@tsyaeger changed the routes to absolute, so you could do something like api/notifications, and similar for above starts from src on frontend

Reply:
Okay I shall change it. Not used to that absolute pathing yet